frontend/src/components/phones/onboarding/RealPhoneSetup.tsx

```diff
@@ -279,9 +287,10 @@ const RealPhoneVerification = (props: RealPhoneVerificationProps) => {
           placeholder={codeEntryPlaceholder}
           required={true}
           maxLength={6}
-          onChange={onChange}
           className={isVerifiedSuccessfully === false ? styles["is-error"] : ""}
+          onChange={onChange}
```

Comment:
If we're stripping out everything that's not a number anyway, might as well guide the user by popping up a number pad on mobile:

```suggestion
          onChange={onChange}
          inputMode="numeric"
```

See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode.